x-pack/test/api_integration/apis/index.js

```diff
@@ -22,7 +22,7 @@ export default function ({ loadTestFile }) {
     loadTestFile(require.resolve('./maps'));
     loadTestFile(require.resolve('./apm'));
     loadTestFile(require.resolve('./siem'));
-    loadTestFile(require.resolve('./code'));
+    // loadTestFile(require.resolve('./code'));
```

Comment:
Since I assume you want people to have the option to enable code, I don’t think you should comment out the tests. Instead, I _think_ it’s possible to explicitly enable code in the test config, so the tests would run as if someone manually enabled it. Thoughts?

Reply:
sorry this is the quickest way that I can think of to disable code in 7.2 only
this PR is not merge for master

other than release purpose I assume no one is going to run test in 7.2 branch?

Reply:
Give that BC is going to be cut today, I suggest we just go with this change
https://github.com/elastic/kibana/pull/38713

Reply:
> Since I assume you want people to have the option to enable code, I don’t think you should comment out the tests. Instead, I _think_ it’s possible to explicitly enable code in the test config, so the tests would run as if someone manually enabled it. Thoughts?

@epixa Ok, take some more time adjusting the PR

Reply:
To be clear, this is just a suggestion. I'm not aware of the context behind this change and what the plans are for 7.2 and whatnot.